core/object/script_language.h

```diff
@@ -371,6 +371,7 @@ class ScriptLanguage : public Object {
 	virtual Vector<StackInfo> debug_get_current_stack_info() { return Vector<StackInfo>(); }
 
 	virtual void reload_all_scripts() = 0;
+	virtual void reload_scripts(const Array &p_script_paths, bool p_soft_reload) = 0;
```

Comment:
```suggestion
	virtual void reload_scripts(const Array &p_scripts, bool p_soft_reload) = 0;
```

It should be like this AIUI, since scripts are passed to this function directly, not their paths.

Reply:
ur so right